src/FormBuilder/ViewModels/FormBuilderViewModel.php

```diff
@@ -50,6 +50,7 @@ public function storageData(int $donationFormId): array
                 'rewriteSlug' => get_post_type_object('give_forms')->rewrite['slug'],
             ],
             'gateways' => $this->getGateways(),
+            'gatewaysGlobalSettings' => $this->getGatewaysGlobalSettings(),
```

Comment:
Since we moved all the scripts and logic out to live with Stripe, let's  move the window data too (where you are enqueuing Stripe into form builder):

```
wp_localize_script( 'givewp-stripe-payment-element-form-builder', 'stripePaymentElementGatewaySettings', $this->getGlobalGatewaySettings());
```
```

Reply:
Sure

Reply:
Resolved: https://github.com/impress-org/givewp/pull/6903/commits/c5a70ef35e2b5f8d1de15e0061bb2d392ab48bf2